istio-telemetry/kiali/values.yaml

```diff
@@ -1,15 +1,14 @@
-#
-# addon kiali
-#
 kiali:
```

Comment:
does the kiali operator create a service for Kiali?  We need that to honor our ingress virtual service routing.

Reply:
Oh, so these comments have some real meaning for something? I can put it back. Yes, there is a Kiali service that the operator creates.

Reply:
done - https://github.com/istio/installer/pull/556/commits/c74e2e59d0ebd7885d022cda76b3e80b1ea187cb